packages/ledger-amino/src/ledgersigner.spec.ts

```diff
@@ -61,7 +62,7 @@ describe("LedgerSigner", () => {
   let transport: Transport;
 
   beforeAll(async () => {
-    if (simappEnabled()) {
+    if (simappEnabled() && ledgerEnabled()) {
```

Comment:
Wait a second, why does this block require Ledger? Souldn't this go into

```ts
  beforeEach(async () => {
    transport = await createTransport();
  });
```

Reply:
Yep. Weird that this change stopped the test failure for me though.